v2/internal/config/vars_test.go

```diff
@@ -80,3 +80,55 @@ func Test_Cloud_ResourceManagerAudienceSet(t *testing.T) {
 	g.Expect(cld.Services[cloud.ResourceManager].Endpoint).To(Equal(cloud.AzurePublic.Services[cloud.ResourceManager].Endpoint))
 	g.Expect(cld.Services[cloud.ResourceManager].Audience).To(Equal(cfg.ResourceManagerAudience))
 }
+
+func TestValidate(t *testing.T) {
+	t.Parallel()
+	g := NewGomegaWithT(t)
+
+	vOk := config.Values{
+		PodNamespace:            "test-namespace",
+		MaxConcurrentReconciles: 1,
+		OperatorMode:            3,
+		TargetNamespaces:        []string{},
+		DefaultReconcilePolicy:  "detach-on-delete",
+	}
+
+	g.Expect(vOk.Validate()).NotTo(HaveOccurred())
+
+	vKoNamespace := config.Values{
+		PodNamespace: "",
+	}
+
+	// g.Expect(vKoNamespace.Validate().Error()).Error().Should(Equal("missing value for POD_NAMESPACE"))
```

Comment:
minor: Remove commented out assert?

Reply:
This commented out assert is still here.